packages/frontend/src/app/shared/forms/credentials-form/credentials-form.component.scss

```diff
@@ -0,0 +1,41 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 2px;
+  background: white;
+  width: 500px;
+  padding: 30px 100px;
+  box-sizing: border-box;
+  -webkit-box-shadow: 0 0 12px 4px rgba(0, 0, 0, 0.1);
+  box-shadow: 0 0 12px 4px rgba(0, 0, 0, 0.1);
+
+  > * {
+    width: 100%;
+  }
+
+  .mat-form-field {
+    margin: 10px 0;
+  }
+
+  .mat-icon {
+    cursor: pointer;
+  }
+
+  .header {
+    margin-bottom: 20px;
+    font-size: 1.6rem;
+    text-align: center;
+    opacity: .5;
+  }
+}
+
+:host ::ng-deep p {
```

Comment:
Is it possible to style button and paragraph without using :host and ::ng-deep? Did you try?

Reply:
https://dev.to/jwp/angular-10-ng-deep-and-host-pseudo-classes-1kbg

Reply:
Ok, I'll try something else